protocol/messenger.go

```diff
@@ -3544,8 +3544,16 @@ func (m *Messenger) handleRetrievedMessages(chatWithMessages map[transport.Filte
 				}
 
 				senderID := contactIDFromPublicKey(publicKey)
+				ownID := contactIDFromPublicKey(m.IdentityPublicKey())
 				m.logger.Info("processing message", zap.Any("type", msg.Type), zap.String("senderID", senderID))
 
+				if senderID == ownID {
```

Comment:
Can it be combined in one statement?

Reply:
Weell yes, but I left them separate intentionally, because there will probably be more message types to skip later.

Related to https://github.com/status-im/status-go/issues/3719.